dasharo-compatibility/326-dasharo-tools-suite.md: rename file, add template for new test cases (#340)
Dasharo/docs: docs/unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md

**Test steps**

1. Plug the USB stick with DTS into the USB slot on the DUT.
1. Power on the DUT.
1. Hold the `BOOT_MENU_KEY` to enter the UEFI Boot Menu.
1. Select the USB stick with DTS using the arrow keys and press `Enter`.

**Expected result**

After a while, the DTS menu should appear.

## DTS002.001 DTS option Creating Dasharo HCL report works correctly

**Test description**

This test aims to verify that the option `Dasharo HCL report` in the DTS menu
properly creates the report.

**Test configuration data**

1. `FIRMWARE` = Dasharo

**Test setup**

1. Proceed with the
    [Test cases common documentation](#test-cases-common-documentation) section.

**Test steps**

1. Plug the USB stick with DTS into the USB slot on the DUT.
1. Power on the DUT.
1. Hold the `BOOT_MENU_KEY` to enter the UEFI Boot Menu.
1. Select the USB stick with DTS using the arrow keys and press `Enter`.
1. Wait for `Enter an option:`.
1. Type in `1` and press `Enter`.
1. Wait for the question: Do you want to support Dasharo development by sending
    us logs with hardware configuration?
1. Type in `y` and press Enter.

**Expected result**

> The report should be generated, the whole process may take a few minutes.

Reviewer: At this point, we should add information, that the test case ends with success if in the summary we could read that all calls exited without errors and the report has been sent to the cloud.

Example summary output:

```bash
SUMMARY
=======

 > All Curl calls exited without errors
 > Attempt to send completed > <report_name>.tar.gz
Thanks you for supporting Dasharo!
```

## DTS003.001 DTS option power-off DUT works correctly

**Test description**

This test aims to verify that the option `Power off system` in the DTS menu
turns off the DUT.

**Test configuration data**

1. `FIRMWARE` = Dasharo

**Test setup**

1. Proceed with the
    [Test cases common documentation](#test-cases-common-documentation) section.

**Test steps**

1. Plug the USB stick with DTS into the USB slot on the DUT.
1. Power on the DUT.
1. Hold the `BOOT_MENU_KEY` to enter the UEFI Boot Menu.
1. Select the USB stick with DTS using the arrow keys and press `Enter`.
1. Wait for `Enter an option:`.
1. Type in `10` and press `Enter`.

**Expected result**

> The DUT should be turned off.

Reviewer: ```suggestion
The DUT should be turned off without any complications.
```

## DTS004.001 DTS option reboot DUT works correctly

**Test description**

This test aims to verify that the option `Reboot system` in the DTS menu reboots
the DUT.

**Test configuration data**

1. `FIRMWARE` = Dasharo

**Test setup**

1. Proceed with the
    [Test cases common documentation](#test-cases-common-documentation) section.

**Test steps**

1. Plug the USB stick with DTS into the USB slot on the DUT.
1. Power on the DUT.
1. Hold the `BOOT_MENU_KEY` to enter the UEFI Boot Menu.
1. Select the USB stick with DTS using the arrow keys and press `Enter`.
1. Wait for `Enter an option:`.
1. Type in `11` and press `Enter`.

**Expected result**

> The DUT should be restarted.

Reviewer: ```suggestion
The DUT should be rebooted without any complications.
```

1. Type in `9` and press `Enter`.

**Expected result**

1. Information about entering the shell and how to exit should be displayed.
1. Shell command input should be activated.

Example output:

```bash
Entering shell, to leave type exit and press Enter or press LCtrl+D

bash-5.1#
```

## DTS006.001 Flash device from DTS shell by using flashrom works correctly

**Test description**

This test aims to verify whether is the possibility to flash the DUT firmware by
using flashrom in `DTS Shell`.

**Test configuration data**

1. `FIRMWARE` = Dasharo

**Test setup**

1. Proceed with the
    [Test cases common documentation](#test-cases-common-documentation) section.
1. `{PATH}` = individual path to a specific binary.

**Test steps**

1. Plug the USB stick with DTS into the USB slot on the DUT.
1. Power on the DUT.
1. Hold the `BOOT_MENU_KEY` to enter the UEFI Boot Menu.
1. Select the USB stick with DTS using the arrow keys and press `Enter`.
1. Wait for `Enter an option:`.
1. Type 9 and click `Enter` to launch Shell.
1. Run the following command to obtain `coreboot.rom` binary:

    ```bash
    wget https://3mdeb.com/open-source-firmware/{PATH} -O /tmp/coreboot.rom
    ```

>     > This is not only way to obtain binary. For example you can use `scp`.

Reviewer: ```suggestion
    > The above-described command is not the only way to obtain binary. For example, `scp` command might be used, too.
```

1. Run the following command to flash the firmware:

    ```bash

>     flashrom -p internal -w /tmp/coreboot.rom

Reviewer: Are You sure about this command? We are going to flash all of the flash chip?

Author: I added information on where to find the exact command for each platform.
